lib/docker-sync/sync_strategy/native_osx.rb

```diff
@@ -23,7 +23,11 @@ def initialize(sync_name, options)
         if @options.key?('image')
           @docker_image = @options['image']
         else
-          @docker_image = 'eugenmayer/unison:2.51.3-4.12.0-AMD64'
+          if RUBY_PLATFORM.include? "arm"
```

Comment:
we should put this detection into a tool method that is more semantic and can be reused on both places.

i assume https://github.com/EugenMayer/docker-sync/blob/master/lib/docker-sync/environment.rb makes most  sense?